frontend/src/components/accountdetails/AccountDetailsPage.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import {Link} from 'react-router-dom'
+import AccountDetailsTable from './AccountDetailsTable'
+
+export default class AccountDetailsPage extends React.Component {
```

Comment:
You have to write a spec with tests for this component. The prompt call will be hard to test but you can test: 
 * what happens when user clicks on the account history button 
 * what happens when user clicks on the delete account button 
 * account details are not rendered when they are not set 
 * account details are rendered 
 * loading message is shown  when loading starts 
 * loading message is hided when loading completes and account details are shown